tools/wptrunner/wptrunner/browsers/android_weblayer.py

```diff
@@ -89,8 +89,8 @@ def __init__(self, logger, binary,
                  symbols_path=None):
         """Creates a new representation of Chrome.  The `binary` argument gives
         the browser binary to use for testing."""
-        super(WeblayerShell, self).__init__(logger,
```

Comment:
Are these changes a necessary part of this change? `WeblayerShell` is still around (doesn't appear elsewhere in the diff) so maybe this is a no-op change? Or was it necessary in some place and this is for consistency?

Reply:
Alright, let's leave it in!